service/lib/dinstaller/manager.rb

```diff
@@ -141,6 +142,10 @@ def software
       @software ||= Software.instance.tap { |s| s.logger = @logger }
     end
 
+    def network
+      @network ||= Network.instance.tap { |s| s.logger = @logger }
```

Comment:
as said also in @joseivanlopez PR. I think we should consider global logger. This is quite annoying to repeat it and also it would be nice to have it automatic.

Reply:
Actually, we could use a mixing for that.